icechunk-python/tests/run_xarray_backends_tests.py

```diff
@@ -0,0 +1,40 @@
+import contextlib
+import os
+import tempfile
+
+import pytest
+
+import zarr
+from icechunk import IcechunkStore, StorageConfig
+from xarray.tests.test_backends import (
+    ZarrBase,
+    default_zarr_version,  # noqa: F401; needed otherwise not discovered
+)
+
+
+@pytest.mark.skipif(
+    not os.environ.get("ICECHUNK_XARRAY_BACKENDS_TESTS", None),
+    reason="skipping Xarray backends tests",
+)
+class TestIcechunkStore(ZarrBase):
+    @contextlib.contextmanager
+    def create_zarr_target(self):
+        if zarr.config.config["default_zarr_version"] == 2:
+            pytest.skip("v2 not supported")
+        with tempfile.TemporaryDirectory() as tmpdir:
+            yield IcechunkStore.create(StorageConfig.filesystem(tmpdir))
```

Comment:
Does running this take a long time? I wonder if it would be valuable to test this against file, mem and minio storage instances.